design/CreateExperiment.md

```diff
@@ -11,6 +11,10 @@ tuned.
   Stay tuned
 * namespace \
   Stay tuned
+* performance_profile \
+  Performance profile is a setting which specifies the SLO related information like function variables and aggregration functions. Here we specify the name of already existing Performance profile. If we create a new performance profile we need to map it with appropriate Recommendation Engine Module for Kruize to generate recommendations. \
```

Comment:
Please change this to 

`performance_profile` is a pre-defined performance objective that is baked into Kruize. `performance_profile` consists of two parts
1. A yaml file defines the metrics to be monitored and the queries used to derive those metrics.
2. Source code that defines the performance objective using the metrics provided

Currently we have the following pre-defined profiles
| Profile Name | yaml | Associated Source | Comments |
| resource-optimization-openshift | [resource-optimization-openshift yaml](https://github.com/kruize/autotune/blob/master/manifests/autotune/performance-profiles/resource_optimization_openshift.yaml) | [source](https://github.com/kruize/autotune/blob/master/src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfileInterface/ResourceOptimizationOpenshiftImpl.java) | This is used for Remote Monitoring  Usecase |
| resource-optimization-local-monitoring | [resource-optimization-openshift yaml](https://github.com/kruize/autotune/blob/master/manifests/autotune/performance-profiles/resource_optimization_local_monitoring.yaml) | [source](https://github.com/kruize/autotune/blob/master/src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfileInterface/ResourceOptimizationOpenshiftImpl.java) | This is used for Remote Monitoring  Usecase |
| default | NA | [source](https://github.com/kruize/autotune/blob/master/src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfileInterface/DefaultImpl.java) | This is applicable to all to Autotune Usecases |

Reply:
Thanks for the review @dinogun I have made the changes as suggested.